docs/server/installation.rst

```diff
@@ -312,19 +312,6 @@ the database and populate it with initial data. This is done by executing the
   (env) $ pootle initdb
 
 
-.. _installation#admin_user:
-
-Creating an admin user
-----------------------
```

Comment:
This can probably be circumvented by setting an email when creating the user on `initdb`. But that will happen in the future so I prefer to land this and focus on more pressing tasks.